index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>togoville jazz festival homepage</title>
+</head>
+<body>
+     
+    <header>
+        <ul class="header-links">
+            <li><a href="#" class="social-link fa fa-twitter"></a></li>
+            <li><a href="#" class="social-link fa fa-facebook"></a></li>
+            <li><a class="lato-font-black" href="#">English</a></li>
+            <li><a class="lato-font-black" href="#">My page</a></li>
+            <li><a class="lato-font-black" href="#">Logout</a></li>
+        </ul>
+    </header>
+    
+    <main> 
+        <nav class="nav-bar">
+            <div class="hamburger-class">
+                <a href="#" ><img src="./img/iconmonstr-menu-1-240.png" alt="menu button"></a>
+            </div>
+            <ul class="menu-bar">
+                <li class="menu-logo" ><a href="./index.html"><img src="./img/level production icon 3.png" alt="level production icon"></a></li>
+                <li><a class="lato-font-black aboutme-link" href="./aboutme.html">About</a></li>
+                <li><a class="lato-font-black" href="#program">Program</a></li>
+                <li><a class="lato-font-black" href="#">Join</a></li>
+                <li><a class="lato-font-black" href="#sponsor">Sponsor</a></li>
+                <li><a class="lato-font-black" href="#">News</a></li>
+                <li class="campaign"><a class="lato-font-red" href="#">LP campaign</a></li>
+            </ul>
+        </nav> 
+        
+        <section class="headline">
+            <p class="lato-font-red" id="greeting">Hello! Music lovers</p>
+            <h1 class="lato-font-red">TOGOVILLE JAZZ <br> JAZZ FESTIVAL <br> 2022</h1>
+            <p class="lato-font-black" id="introduction">A lively festival celebrating popularizing, sharing of musical knowledge and discussing the political and cultural aspects that emerge from it</p>
+            <p class="lato-font-black" id="event-date">2022.05.15(TUE) ~ 27(SUN)</p>
+            <p class="lato-font-black">@French Institute of Togo, Goethe Institute,  and more</p>
+
+        </section>
+
+        <section class="main-program" id="program">
```

Comment:
- [ ] In my opinion, there is not enough color contrast between the background image and the color of the cards. I think it would be great if you use a lighter color to match the template. 🙂️

**Your design**

![Screenshot from 2023-02-09 13-27-20](https://user-images.githubusercontent.com/17243325/217798928-233ef53b-cf82-4894-b18f-a2b7fc73d16d.png)

**Template**

![Screenshot from 2023-02-09 13-26-59](https://user-images.githubusercontent.com/17243325/217799038-29ba3f27-b30f-4035-9364-ee0c21fa5938.png)